app/scripts/controllers/provisionedServices.js

```diff
@@ -0,0 +1,265 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:ProvisionedServicesController
+ * @description
+ * # ProjectController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('ProvisionedServicesController', function ($routeParams,
+                                                         $scope,
+                                                         APIService,
+                                                         BindingService,
+                                                         Constants,
+                                                         DataService,
+                                                         ProjectsService,
+                                                         ResourceAlertsService,
+                                                         $filter,
+                                                         LabelFilter,
+                                                         Logger) {
+
+    //var overview = this;
+    //var limitWatches = $filter('isIE')() || $filter('isEdge')();
+    //var DEFAULT_POLL_INTERVAL = 60 * 1000; // milliseconds
+
+    // Enable service catalog features if the new experience is enabled and the
+    // servicecatalog.k8s.io resources are available.
+    var SERVICE_CATALOG_ENABLED =
+      _.get(Constants, 'ENABLE_TECH_PREVIEW_FEATURE.service_catalog_landing_page') &&
+      APIService.apiInfo({ group: 'servicecatalog.k8s.io', resource: 'serviceclasses' }) &&
+      APIService.apiInfo({ group: 'servicecatalog.k8s.io', resource: 'instances' }) &&
+      APIService.apiInfo({ group: 'servicecatalog.k8s.io', resource: 'bindings' });
```

Comment:
Nice! That was one of the questions, can we centralize it seems silly to keep repeating it, I'll take a look at his PR